docs/sops/msft-test-test-tenant-access.md

```diff
@@ -0,0 +1,284 @@
+   - **Click "Accept invitation"** in the email
+
+### 2. Access Level
+
+Once invited, you will typically receive **Contributor** access to:
+- **ARO HCP E2E - Staging** (Subscription ID: `99399281-00a2-4b39-bb3d-b2645bbbdb93`)
+- **ARO HCP E2E** (Subscription ID: `403d9de9-132b-4974-94a5-5b78bdfa191e`)
+
+### 3. Accepting the Invitation
+
+**Important**: The initial invitation acceptance may require specific network access:
+
+- **Option 1: SAW (Secure Admin Workstation)** - Recommended for initial invitation acceptance
+  - Open the invitation link in your SAW device by logging in with your b-account
+  - Set up multi-factor authentication (MFA) for this tenant in the Microsoft Authenticator app on your phone
+  - This ensures proper authentication through Microsoft's secure channels
+
+- **Option 2: VPN Connection**
+  - You may also be able to accept this invitation by connecting to the MSFT VPN, but if this fails please use your SAW instead
+
+**Note**: After the initial invitation is accepted (typically via SAW), subsequent access from your local development machine may work without VPN. However, this varies by user configuration and network setup. If you encounter authentication issues, try:
+1. Connecting to VPN
+2. Using SAW for the operation
+3. Contacting the Service Lifecycle team for assistance
+
+### 4. Verify Access
+
+Once approved, verify your access:
+
+1. **Login to Azure Portal**
+   ```bash
+   az login --tenant 93b21e64-4824-439a-b893-46c9b2a51082
+   ```
+
+2. **List accessible subscriptions**
+   ```bash
+   az account list --output table
+   ```
+   
+   You should see:
+   ```
+   Name                       CloudName    SubscriptionId                        State    IsDefault
+   -------------------------  -----------  ------------------------------------  -------  -----------
+   ARO HCP E2E - Staging      AzureCloud   99399281-00a2-4b39-bb3d-b2645bbbdb93  Enabled  False
+   ARO HCP E2E                AzureCloud   403d9de9-132b-4974-94a5-5b78bdfa191e  Enabled  False
+   ```
+
+3. **Set the subscription**
+   ```bash
+   az account set --subscription "ARO HCP E2E - Staging"
+   ```
+
+4. **Verify you can list resources**
+   ```bash
+   az group list --output table
+   ```
+
+5. **Verify tenant ID**
+   ```bash
+   az account show --query tenantId -o tsv
+   ```
+   
+   Expected output: `93b21e64-4824-439a-b893-46c9b2a51082`
+
+## CI/CD Service Principal Configuration
+
+**✅ Status**: Service principals and credentials for the MSFT Test Test tenant have **already been configured** for ARO HCP CI/CD.
+
+### Current Setup
+
+The ARO HCP project uses service principals to authenticate OpenShift CI (Prow) jobs with the MSFT Test Test tenant. The service principals were created using the setup script at [`dev-infrastructure/openshift-ci/create-openshift-release-bot-msft-test.sh`](../../dev-infrastructure/openshift-ci/create-openshift-release-bot-msft-test.sh), and the credentials were stored in OpenShift CI Vault.
```

Comment:
Someone should really look into OIDC federation between this tenant and the OCP CI clusters - you should be able to mint MSI.